src/panels/CharmsPanel/CharmsPanel.test.tsx

```diff
@@ -39,27 +39,41 @@ describe("CharmsPanel", () => {
   });
 
   it("renders the correct number of charms", () => {
-    renderComponent(<CharmsPanel />, { path, url, state });
+    renderComponent(<CharmsPanel onCharmURLChange={() => {}} />, {
+      path,
+      url,
+      state,
+    });
     expect(screen.getAllByRole("radio").length).toBe(2);
   });
 
   it("next button is disabled when no charm is selected", () => {
-    renderComponent(<CharmsPanel />, { path, url, state });
+    renderComponent(<CharmsPanel onCharmURLChange={() => {}} />, {
+      path,
+      url,
+      state,
+    });
     expect(screen.getByRole("button", { name: "Next" })).toBeDisabled();
   });
 
   it("next button is enabled when a charm is selected", () => {
-    renderComponent(<CharmsPanel />, { path, url, state });
+    renderComponent(<CharmsPanel onCharmURLChange={() => {}} />, {
```

Comment:
This will currently make no difference to these tests, but the convention is to use `jest.fn()` to provide functions for props in tests. The benefit will be that if the function type changes (e.g. a return type is added) the `() => {}` will no longer satisfy the type, but `jest.fn()` will.

Reply:
I never thought about using `jest.fn()` instead of `() => {}` from a "maintainability" standpoint. Thanks for the remark!